swift-pipeline

```diff
@@ -256,6 +308,7 @@ if __name__ == "__main__":
                 recreate_single_figure(
                     plot=plot,
                     line_data=line_data,
+                    colours=args.colours,
```

Comment:
I suppose this would need to come along with additional changes to the velociraptor library?